### PR TITLE
Add test for trailing whitespace in file entries of InitialWorkDir

### PR DESCRIPTION
**Pull request by anton-khodak**
Inspiration: https://github.com/common-workflow-language/cwltool/issues/247

### CODE REVIEW

File: v1.0/conformance_test_v1.0.yaml [hunk 1]
```diff
@@ -462,6 +462,12 @@
   doc: |
     Test InitialWorkDirRequirement with expression in filename.
 
+- job: v1.0/string-job.json
+  output:
+    out: "CONFIGVAR=hello\n"
```
Comment (mr-c) [inline v1.0/conformance_test_v1.0.yaml]: There's not `\n` in YAML :-) How about a regular `File` output and a checksum?
